### PR TITLE
Add event faq & cfp

### PR DESCRIPTION
**Pull request by tracymiranda**

### CODE REVIEW

**Comment by tracymiranda** [comment]
Screenshot 2024-07-10 at 8 47 03 AM

**Comment by tracymiranda** [comment]
CFP:
Screenshot 2024-07-10 at 8 47 20 AM

**Review by jiggoha — APPROVED** [review]

File: content/application/summit.html [hunk 1]
```diff
@@ -148,3 +151,34 @@ <h2 class="headline4 gmb-4">Agenda & further details coming soon!</h2>
...
+        You can easily get to the venue via tube, bike or foot. It is a ~2 minute walk from <a href="https://maps.app.goo.gl/aRSybpgFsRSQ7CPG6">Kings Cross</a> and <a href="https://maps.app.goo.gl/MpPsnuoDUyEK4Yy9A">St Pancras</a> stations. 
+      </p>
+      <h3 class="headline5 gpt-2 gmb-2">Airport</h3>
+      <p class="gmb-2">
+        Airport
+If you haven’t visited London in a while (or before!), we have exciting news, the Elizabeth line is open! This gives a couple of choices for getting to the venue either with the Elizabeth line or Piccadilly: <a href="https://maps.app.goo.gl/yvJrzmcqGe6ZqfuD6">Heathrow to Google 6PS</a> (~60 minutes).
+      </p>
+      <h3 class="headline5 gpt-2 gmb-2">Room Block</h3>      
+      <p class="gmb-1">
+        We do not have a room block with discounted rooms, but there are plenty of great places to stay nearby, including:
```
Comment (phbnf) [inline content/application/summit.html:169]: I've also heard good reviews of this one: https://maps.app.goo.gl/UdVqc54jujpQkZJC6, it's a good compromise between the two others: much cheaper than the Great Northern, a bit more expensive than EasyHotel, but much closer.

**Review by phbnf — APPROVED** [review]

File: content/application/summit.html (hunk 1 above)
Comment (tracymiranda) [reply content/application/summit.html:169]: Great, it will be good to have a 3rd option - will add it in.

**Pull request merged**